.github/workflows/update-runner-version.yaml

```diff
@@ -0,0 +1,36 @@
+name: update runner version
+on:
+  pull_request:
+    paths:
+      - "runner-images/RUNNER_VERSION"
+  push:
+    branches:
+      - "main"
+    paths:
+      - "runner-images/RUNNER_VERSION"
+jobs:
+  get-versions:
+    name: Get versions for runner image
+    runs-on: ubuntu-24.04
+    outputs:
+      meows_version: ${{ steps.meows_version.outputs.version }}
+      runner_version: ${{ steps.runner_version.outputs.version }}
+    steps:
+      - uses: actions/checkout@v5
+      - name: Get meows version
+        id: meows_version
+        run: |
+          meows_version="$(awk -F'"' '/Version =/ {print $2}' constants.go)"
+          echo version=$meows_version >> $GITHUB_OUTPUT
+      - name: Get runner version
+        id: runner_version
+        run: |
+          echo version="$(cat ./runner-images/RUNNER_VERSION)" >> $GITHUB_OUTPUT
+  runner-images:
```

Comment:
[IMO]
It's preferable to use a "verb + description" format when naming jobs.

> The name attribute jobs set must be provided and follow the pattern “Verb description”. Start with a verb so there is a clear expectation of what the step is for.
> https://nimblehq.co/compass/development/code-conventions/github-actions/

For example:
- runner-images -> push-runner-images
- released-versions -> get-released-versions